src/markdown/components/windowed.md

```diff
@@ -44,11 +44,11 @@ This demo generates the data after you click the overlay. Generating 500000 rows
 | `load(Array)` | Loads an array of data. |
 | `loaded()` | Called after the load function has been called. |
 | `getRows()` | Returns an array of all rows that are currently loaded. |
-| `getRow(Number)` | Get a row of data (returns an awaitable promise). |
+| `getRow(Number)` | Get a row of data (returns an available promise). |
```

Comment:
`an awaitable promise` is correct spelling here

Reply:
Okay, I will correct that.